pkgs/by-name/st/steam/package.nix

```diff
@@ -89,6 +89,10 @@ let
       export LIBVA_DRIVERS_PATH=/run/opengl-driver/lib/dri:/run/opengl-driver-32/lib/dri
       export VDPAU_DRIVER_PATH=/run/opengl-driver/lib/vdpau:/run/opengl-driver-32/lib/vdpau
 
+      # This is needed on KDE Plasma + Wayland + SDL 2 Stable to find the cursors properly. Already fixed in SDL3 (cursor-shape-v1) and in Gnome.
+      # https://forums.factorio.com/113202
+      export XCURSOR_PATH=$(readlink -f /run/current-system/sw)/share/icons
```

Comment:
Is the `readlink` necessary? Can we point it directly to the symlinked path?

Reply:
My current understanding is that the readlink needs to happen before the game is launched into the FHS wrapped environment, because it sees a different `/run` but the same `/nix`.

I would welcome help figuring out what would work here to not need the readlink.

Just tested and setting it directly `XCURSOR_PATH=/run/current-system/sw/share/icons %command%` makes the bug return:
![image](https://github.com/user-attachments/assets/1f565f11-62ab-40ac-9338-a521268858db)

While using the `.override` steam with no launch options works properly:
![image](https://github.com/user-attachments/assets/275a5f09-a89b-4259-8b1b-52da4ccdfef2)

Reply:
Actually, that draws me to a simple realization: we can just add this folder into what gets copied into the FHS `/run`, right? Where is that?

Reply:
The fhsenv should have the same shared /run, so this is likely an environment issue.